apps/health_campaign_field_worker_app/lib/pages/home.dart

```diff
@@ -198,7 +199,8 @@ class _HomePageState extends LocalizedState<HomePage> {
                       completedSync: () async {
                         Navigator.of(context, rootNavigator: true).pop();
                         await localSecureStore.setManualSyncTrigger(false);
-                        if (context.mounted) {
```

Comment:
Is this covering all cases we have multiple states sync up failed sync down failed, check if there is an inbuilt method to identify rather than using booleans